### ISSUE TITLE
npm start fails with undefined variable

### ISSUE THREAD

**Issue by LongboatAline** [tidepool-org/tideline#333]
Following the README.md, I'm failing at the `npm start` stage with a message that suggests I'm missing some css. Did I overlook something, or is something else amiss?
(as might be subtly obvious, I'm a complete noob to the ways of npm, javascript and everything modern. But they do seem to generate nicer results with these tasks than `sed/awk/sh` would :)

I tried two different branches,`master` and `handle-600-smbg-annotations` as of today with identical results.

I'd be happy for any hint that would get me gping 
```
   ...
  [462] ./~/css-loader!./~/autoprefixer-loader!./~/less-loader!./css/tideline.less 21.1 kB {0} [built]
  [463] ./example/less/example.less 1.18 kB {0} [built]
  [464] ./~/css-loader!./~/autoprefixer-loader!./~/less-loader!./example/less/example.less 11.7 kB {0} [built]
  [465] ./fonts/blip-icons.eot 10.4 kB {0} [built]
  [466] ./fonts/blip-icons.woff 6.11 kB {0} [built]
  [467] ./fonts/blip-icons.ttf 10.1 kB {0} [built]
  [468] ./fonts/blip-icons.svg 7.34 kB {0} [built]
  [469] ./fonts/opensans.woff 20.2 kB {0} [built]
  [470] ./fonts/opensans-bold.woff 20.1 kB {0} [built]
  [471] ./fonts/opensans-italic.woff 21.1 kB {0} [built]
  [472] ./fonts/opensans-bolditalic.woff 20.7 kB {0} [built]
  [473] ./fonts/opensans-light.woff 20.1 kB {0} [built]
  [474] ./fonts/opensans-semibold.woff 20.4 kB {0} [built]
  [475] ./fonts/opensans-lightitalic.woff 21.3 kB {0} [built]
  [476] ./fonts/opensans-semibolditalic.woff 21 kB {0} [built]

ERROR in ./~/css-loader!./~/autoprefixer-loader!./~/less-loader!./example/less/modalsubnav.less
Module build failed: variable @monday is undefined
 @ /Volumes/Users/theuser/src/GitHub/Tidepool/tideline-automated-basal/example/less/modalsubnav.less (line 99, column 24)
 near lines:
       &.monday.active {
         background-color: @monday;
       }
 @ ./example/less/modalsubnav.less 4:14-172
webpack: Failed to compile.
```

**Comment by cbwebdevelopment** [comment]
@LongboatAline Thanks for bringing this to our attention.

I'll get started on a fix for this today, and should have it reviewed and published at some point next week.

**Comment by cbwebdevelopment** [comment]
FIxed in `v0.7.2`
